Makefile

```diff
@@ -50,21 +50,25 @@ test-large: test-large-init  ## Run large test set
 Dockerfile: Dockerfile.env Dockerfile.in
 	poetry run jinja2 Dockerfile.in Dockerfile.env > Dockerfile
 
+.PHONY: poetry-install
```

Comment:
Just a note here (so auto-closing the comment as it's not something to solve): I believe this should be what we do in this makefile: run `make target` and have the makefile call `poetry` for us, rather than doing `poetry run make target`.